server/models.ts

```diff
@@ -0,0 +1,116 @@
+export const CommunityAdmin = sequelize.import('./communityAdmin/model');
+export const CrossrefDepositRecord = sequelize.import('./crossrefDepositRecord/model');
+export const Discussion = sequelize.import('./discussion/model');
+export const Doc = sequelize.import('./doc/model');
+export const Export = sequelize.import('./export/model');
+export const ExternalPublication = sequelize.import('./externalPublication/model');
+export const Member = sequelize.import('./member/model');
+export const Merge = sequelize.import('./merge/model');
+export const Organization = sequelize.import('./organization/model');
+export const Page = sequelize.import('./page/model');
+export const Pub = sequelize.import('./pub/model');
+export const PubAttribution = sequelize.import('./pubAttribution/model');
+export const PubEdge = sequelize.import('./pubEdge/model');
+export const PubManager = sequelize.import('./pubManager/model');
+export const PubVersion = sequelize.import('./pubVersion/model');
+export const PublicPermissions = sequelize.import('./publicPermissions/model');
+export const Release = sequelize.import('./release/model');
+export const Review = sequelize.import('./review/model');
+export const ReviewEvent = sequelize.import('./reviewEvent/model');
+export const Signup = sequelize.import('./signup/model');
+// export const Thread = sequelize.import('./thread/model');
+// export const ThreadAnchor = sequelize.import('./threadAnchor/model');
+// export const ThreadComment = sequelize.import('./threadComment/model');
+// export const ThreadUser = sequelize.import('./threadUser/model');
+
+export const DiscussionNew = sequelize.import('./discussion/modelNew');
+export const Anchor = sequelize.import('./anchor/model');
+export const ReviewNew = sequelize.import('./review/modelNew');
+export const Fork = sequelize.import('./fork/model');
+export const Thread = sequelize.import('./thread/model');
+export const ThreadComment = sequelize.import('./threadComment/model');
+export const ThreadEvent = sequelize.import('./threadEvent/model');
+export const Visibility = sequelize.import('./visibility/model');
+export const VisibilityUser = sequelize.import('./visibilityUser/model');
+
+export const User = sequelize.import('./user/model');
+export const WorkerTask = sequelize.import('./workerTask/model');
+
+export const attributesPublicUser = [
+	'id',
+	'firstName',
+	'lastName',
+	'fullName',
+	'avatar',
+	'slug',
+	'initials',
+	'title',
+	'orcid',
+];
+
+export const includeUserModel = (() => {
+	return ({ attributes: providedAttrs, ...restOptions }) => {
+		const attributes = providedAttrs
+			? // @ts-expect-error ts-migrate(2569) FIXME: Type 'Set<any>' is not an array type or a string t... Remove this comment to see the full error message
+			  [...new Set([...attributesPublicUser, ...providedAttrs])]
+			: attributesPublicUser;
+		return {
+			model: User,
+			attributes: attributes,
+			...restOptions,
+		};
+	};
+})();
```

Comment:
Relevant change. `includeUserModel` was moved into this file due to the odd error I was getting.